src/scripts/unit_tests

```diff
@@ -16,5 +16,9 @@ Run all unit tests.
 if [ "${1:-}" = "--help" ]; then
     usage
 else
-    python -m unittest discover tests
+    if ! [ -x "$(command -v coverage)" ]; then
+	python -m unittest discover tests
+    else
+	coverage run --append --source=rastervision --omit=rastervision/protos -m unittest discover tests
+    fi
```

Comment:
Can you explain what this does? Perhaps a comment would clear it up, but I'm not sure how to use this.

Reply:
It runs the tests with the command `python -m unittest discover tests` if coverage.py is not intalled.  If coverage.py is installed it used the command `coverage run --append --source=rastervision --omit=rastervision/protos -m unittest discover tests` .

Reply:
Ah, gotcha, so coverage will actually run the tests as well.